types/index.d.ts

```diff
@@ -1808,6 +1824,22 @@ export namespace Moralis {
     }
   }
 
+  interface IChains {
+    ETH_MAINET: EChains.ETH_MAINENT;
+    ETH_ROPSTEN: EChains.ETH_ROPSTEN;
+    ETH_GOERLI: EChains.ETH_GOERLI;
+    ETH_KOVAN: EChains.ETH_KOVAN;
+    ETH_RINKBEY: EChains.ETH_RINKBEY;
+    ETH_LOCALDEVCHAIN: EChains.ETH_LOCALDEVCHAIN;
+    POLYGON_MAINNET: EChains.POLYGON_MAINNET;
+    POLYGON_MUMBAI: EChains.POLYGON_MUMBAI;
+    BSC_MAINNET: EChains.BSC_MAINNET;
+    BSC_TESTNET: EChains.BSC_TESTNET;
+    AVAX_MAINNET: EChains.AVAX_MAINNET;
+    AVAX_TESTNET: EChains.AVAX_TESTNET;
+    FANTOM_MAINNET: EChains.FANTOM_MAINNET;
+  }
+
```

Comment:
```suggestion

  let Chains: IChains;
```

Add this to export the Moralis.Chains type correctly